tests/src/Functional/NewsIntegrationTest.php

```diff
@@ -88,9 +88,7 @@ public function testCreateNews() {
     $assert_session->pageTextContains('Example Content');
     $assert_session->pageTextContains('Example Introduction');
     $assert_session->pageTextContains('01/24/2022');
-    $assert_session->responseContains('image-test.png');
```

Comment:
We can't assert the image itself but we can assert that there's the title of the media and its caption rendered.

Reply:
Nice call, I did for Events as well.